packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAdsTableRow/MyAdsTableRow.tsx

```diff
@@ -0,0 +1,225 @@
+import React, { memo, useEffect, useState } from 'react';
+import clsx from 'clsx';
+import { useExchangeRateSubscription } from '@deriv/api';
+import { Button, Text, Tooltip } from '@deriv-com/ui';
+import { useDevice } from '../../../../../hooks';
+import { generateEffectiveRate } from '../../../../../utils/format-value';
+import DeactivateIcon from '../../../../../public/ic-archive.svg';
+import ActivateIcon from '../../../../../public/ic-unarchive.svg';
+import EditIcon from '../../../../../public/ic-edit.svg';
+import DeleteIcon from '../../../../../public/ic-delete.svg';
```

Comment:
another note, see if we have these icons in quill-icons as well

Reply:
as of now, they are not available, will request in the channel